.github/workflows/cli-test-npm.yml

```diff
@@ -17,22 +17,38 @@ jobs:
         uses: actions/setup-node@v2
         with:
          node-version: ${{ matrix.node-version }}
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+         node-version: ${{ matrix.node-version }}
+      - name: Check Out Repo
+        uses: actions/checkout@v2
       -  name: Global - Testing installation of Twilio CLI (1)
          run: |
-          npm make install -g twilio-cli@${{ github.event.inputs.version }}
-          node test/asserts/validate_tests.js
+          npm install -g twilio-cli@${{ github.event.inputs.version }}
+          chmod +x ./.github/scripts/cmds-to-test.sh
+          ./.github/scripts/cmds-to-test.sh
+      
       - name: Global - Testing upgrade Twilio CLI (2)
         run: |
-         npm make install -g twilio-cli@${{ github.event.inputs.version }}
+         npm install -g twilio-cli@${{ github.event.inputs.version }}
             npm install -g twilio-cli
-            node test/asserts/validate_tests.js
-      - name: Local - Testing installation of Twilio CLI (1)
-        run: |
-         npm install twilio-cli
-         node test/asserts/validate_tests.js
-      - name: Local - Testing upgrade Twilio CLI (2)
+            chmod +x ./.github/scripts/cmds-to-test.sh
```

Comment:
Consider using git `--chmod=+x` flag while committing this file to push as an executable to avoid manually changing permissions everytime before execution. Applies everywhere else too

Ref: https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows